hw/top_earlgrey/ip/ast/rtl/sys_osc.sv

```
@@ -57,20 +57,18 @@ always begin
   jitter = sys_jen_i ? $urandom_range(2000, 0) : 0;
   #((SysClkPeriod+jitter)/2000) clk = ~clk && en_osc;
 end
-
-assign sys_clk_o = clk;
 `else  // of SYNTHESIS
-localparam prim_pkg::impl_e Impl = `PRIM_DEFAULT_IMPL;
-
 // SYNTHESUS/VERILATOR/LINTER/FPGA
 ///////////////////////////////////////
-logic clk, en_osc, en_osc_re, en_osc_fe;
+localparam prim_pkg::impl_e Impl = `PRIM_DEFAULT_IMPL;
+logic clk, clk_n, en_osc, en_osc_re, en_osc_fe;
 // TODO: add sys_jen_i
 
 assign en_osc_re = vcore_pok_h_i && sys_en_i;
+assign clk_n = !clk;
```

dito

---

I did that to silence error in AscenLint that compain on the use of negedge clock...
I need to see It the tool clear this error!